package.json

```diff
@@ -54,8 +54,8 @@
     "lint": "eslint . --max-warnings=0",
     "local:web": "npm run dev --prefix src/app",
     "local": "ts-node --cwdMode --transpileOnly src/main.ts",
-    "preversion": "[ \"$(git rev-parse --abbrev-ref HEAD)\" = \"main\" ] || (echo \"Error: Must be on main branch to version\" && exit 1) && git pull origin main && git checkout -b \"chore/bump-version-$npm_package_version\"",
-    "postversion": "npm run citation:generate && git add CITATION.cff && git commit --amend --no-edit && gh pr create --title \"chore: bump version $npm_package_version\" --body \"\"",
+    "preversion": "[ \"$(git rev-parse --abbrev-ref HEAD)\" = \"main\" ] || (echo \"Error: Must be on main branch to version\" && exit 1) && git pull origin main",
+    "postversion": "git checkout -b \"chore/bump-version-$npm_package_version\" && npm run citation:generate && git add CITATION.cff && git commit --amend --no-edit && gh pr create --title \"chore: bump version $npm_package_version\" --body \"\"",
```

Comment:
This won't work because `npm version` automatically creates a commit when you run the the command.